### ISSUE TITLE
add function

### ISSUE THREAD

**Issue by PasaOpasen** [timmahrt/pysle#9]
Please add some function like:

```python

transcribe('text') # 'tekst'

transcribe('my friend') # "mai frent"

```

**Comment by timmahrt** [comment]
I've implemented something.  If you're ok with it, I'll make a release.  Please let me know what you think.

```
sentence = "do you want another pumpkinseed"
phoneList = isletool.transcribe(isleDict, sentence, 'longest')
print(phoneList)

> 'du ju want ənʌðəɹ pʌmpkɪnsid'
```
https://github.com/timmahrt/pysle/pull/10

**Comment by PasaOpasen** [comment]
Yes it's all I need, I will wait ur release

**Comment by timmahrt** [comment]
Ok, I've released version 2.1.0.  After upgrading, please let me know if you have any feedback.  If its good, please close this issue.  Thanks!
